.github/workflows/deploy.yaml

```
@@ -23,15 +21,27 @@ jobs:
         run: |
           curl -fsSL https://d2lang.com/install.sh | sh -s --
-      - name: Install, build, and upload your site
-        uses: withastro/action@v4
-        # with:
-        # path: . # The root location of your Astro project inside the repository. (optional)
-        # node-version: 20 # The specific version of Node that should be used to build your site. Defaults to 20. (optional)
-        # package-manager: pnpm@latest # The Node package manager that should be used to install dependencies and build your site. Automatically detected based on your lockfile. (optional)
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          enable-corepack: true
```

actionlint (1.7.11)

[error] input "enable-corepack" is not defined in action "actions/setup-node@v4". available inputs are "always-auth", "architecture", "cache", "cache-dependency-path", "check-latest", "mirror", "mirror-token", "node-version", "node-version-file", "registry-url", "scope", "token"

(action)